docs/guide/task-runner.md

```diff
@@ -13,7 +13,9 @@ you can run and test your tasks with `dotnet husky run` command. Once you are su
 e.g
 
 ``` shell:no-line-numbers:no-v-pre
-dotnet husky add pre-commit -c "dotnet husky run"
+stages_files=$(git diff --staged --name-only --no-ext-diff --diff-filter=AM | tr '\n' ';' | sed 's/;$//')
```

Comment:
Since the whole idea behind husky.net is simplicity and avoiding the custom commands, this change is unnecessary, it might confuse the new users.

Reply:
currently we don't have a separate page for these arguments, we should add one in the documentation later.